src/parser/transformer.cpp

```diff
@@ -756,6 +758,18 @@ std::unique_ptr<ParsedExpression> Transformer::transformListLiteral(
     return listCreation;
 }
 
+std::unique_ptr<ParsedExpression> Transformer::transformStructLiteral(
+    CypherParser::OC_StructLiteralContext& ctx) {
+    auto structPack =
+        std::make_unique<ParsedFunctionExpression>(common::STRUCT_PACK_FUNC_NAME, ctx.getText());
+    for (auto& structField : ctx.oC_StructField()) {
+        auto structExpr = transformExpression(*structField->oC_Expression());
+        structExpr->setAlias(transformSymbolicName(*structField->oC_SymbolicName()));
```

Comment:
We probably need to discuss how to access a field from a struct. That part is currently missing and might be tricky to do.